assets/styles/slug-pages.scss

```diff
@@ -81,3 +86,8 @@
   max-width: 100%;
 }
 
+:deep(.ftva.page-anchor .dropdown-button),
```

Comment:
@farosFreed I removed these two page-anchor styles in my general-content page ticket since the update I made in the component library makes them unneeded.